include/power_grid_model/calculation_parameters.hpp

```diff
@@ -59,6 +59,17 @@ struct BranchMathOutput {
     ComplexValue<sym> i_t;
 };
 
+// fault math calculation parameters and math output
+struct FaultCalcParam {
+    Idx math_fault_object{-1};
+    DoubleComplex y_fault;
+};
+
```

Comment:
we need to define a `ShortCircuitMathInput`, which is actually a collection of faults. This will be the math input.

```c++
struct ShortCircuitMathInput {
    std::vector<FaultCalcParam> faults;
};
```